setup.py

```diff
@@ -40,6 +40,7 @@
             "course_bookmarks = openedx.features.course_bookmarks.plugins:CourseBookmarksTool",
             "course_updates = openedx.features.course_experience.plugins:CourseUpdatesTool",
             "course_reviews = openedx.features.course_experience.plugins:CourseReviewsTool",
+            "verified_upgrade = courseware.course_tools:VerifiedUpgradeTool",
```

Comment:
Don't forget to bump the version number when introducing a new entry point. This is needed so that the new package will be installed for folks if they already have the previous version.

Reply:
Ah, good catch, thanks!